### ISSUE TITLE
Software Renderer...

### ISSUE THREAD

**Issue by godzil21** [libretro/reicast-emulator#44]
I don't know if it is possible but I'm trying to create a libretro frontend in C#  with Visual Studio and to make the tests I need to execute the core with software renderer, but I don't find the option .

I have tried to put HW_RENDER in the set_environment callback to false but doesn't seem to work. 

Any idea? hoping I won't have to recompile the entire core without open_gl lib only to use the software renderer. 

Thanks a lot guys!

**Comment by Alcaro** [comment]
No, [you must recompile](https://github.com/libretro/reicast-emulator/blob/master/core/libretro/libretro.cpp#L619).

If you're new to libretro, this is a bad starting core; you have to implement a lot of extensions before anything happens. Start with Snes9x instead.

**Comment by godzil21** [comment]
Thanks Alcaro, that was my guess...
